### DISCUSSION TITLE
Error on import CardPayment

### DISCUSSION THREAD

**Discussion by checomendoza** [mercadopago/sdk-react#49] [Ideas]
I created a small project in NEXTjs with CardPayment component, but when run dev enviroment, give this error
```
import initMercadoPago from './mercadoPago/initMercadoPago';
^^^^^^
SyntaxError: Cannot use import statement outside a module
```

This is the code:

```
import React from 'react'
import { CardPayment } from '@mercadopago/sdk-react'

const Index = () => {
	const AMOUNT = 150

	return (
		<div>
			<h1>CHECKOUT MERCADOPAGO</h1>
			<CardPayment
				initialization={{ amount: AMOUNT }}
				onSubmit={async (param) => {
					console.log(param)
				}}
			/>
		</div>
	)
}
export default Index
```

How I could resolve this error?

**Comment by AleFossati** [comment]
Hello, I've tried to create a NextJS project with Card Payment brick and it's working fine.
Can you share a codepen, or a video showing how the issue occurs?

That's my code:

```tsx
'use client';

import { TCardPayment } from '@mercadopago/sdk-react/bricks/cardPayment/type';
import styles from './page.module.css'
import { initMercadoPago, CardPayment } from '@mercadopago/sdk-react'

initMercadoPago('YOUR-PUBLIC-KEY')

export default function Home() {

  const setup: TCardPayment = {
    onSubmit: async (formData) => console.log(formData),
    initialization: {
      amount: 1000
    }
  }

  return (
    <main className={styles.main}>
      <div className={styles.brick}>
        <CardPayment {...setup} />
      </div>
    </main>
  )
}
```

**Comment by emmanuel-salvo** [comment]
Unfortunaly, I have the same problem here. I tried to implement the "mercadopago/sdk-react" module to my already created Next app and it threw me the same error.

**Reply by emmanuel-salvo** [reply]
*UPDATE*: I don't know what happened, but if I go to the page that has the Payment component with a Next's Link Component (without making the browser to navigate) it doesn't throw me any error, but if I instead refresh the browser on the page that has the Payment component it throws me the error.

**Reply by AleFossati** [reply]
Hello @emmanuel-salvo, can you show share your code, so I can try it?
In my tests here, it's working fine.

**Reply by ghost** [reply]
@emmanuel-salvo hello! any news on this?
